Project.toml

```diff
@@ -13,6 +13,7 @@ DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 DynamicPPL = "366bfd00-2699-11ea-058f-f148b4cae6d8"
 EllipticalSliceSampling = "cad2338a-1db2-11e9-3401-43bc07c9ede2"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+GpABC = "e850a1a4-d859-11e8-3d54-a195e6d045d3"
```

Comment:
Let's make this an optional dependency via using `Requires.jl`.

Reply:
Done :+1: